src/commands/force/user/password/generate.ts

```diff
@@ -35,6 +50,11 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
   public async run(): Promise<PasswordData[] | PasswordData> {
     this.usernames = (this.flags.onbehalfof as string[]) ?? [this.org.getUsername()];
 
+    const passwordCondition: PasswordConditions = {
+      length: this.flags.length ? asNumber(this.flags.length) : 13,
```

Comment:
`asNumber` takes a second parameter to be the default value. This could simplify to `{ length: asNumber(this.flags.length, 13)`